common/constants/shared.ts

```diff
@@ -85,7 +85,7 @@ export const SAVED_OBJECT = '/object';
 // Color Constants
 export const PLOTLY_COLOR = [
   '#3CA1C7',
-  '#8C55A3',
```

Comment:
Why removing #8C55A3?

Reply:
To change the colors in the chart I assumed that I should override the default purple color used with the green value mentioned in the Opensearch UI docs. Should I instead add the green as another color and then change the direct call (e.g. PLOTLY_COLOR[1] -> PLOTLY_COLOR[10])?

Reply:
Sounds good, I'll revert this change